llvm/include/llvm/CodeGen/GlobalISel/MachineIRBuilder.h

```diff
@@ -1267,7 +1267,7 @@ class MachineIRBuilder {
   /// \return a MachineInstrBuilder for the newly created instruction.
   MachineInstrBuilder buildICmp(CmpInst::Predicate Pred, const DstOp &Res,
                                 const SrcOp &Op0, const SrcOp &Op1,
-                                std::optional<unsigned> Flgs = std::nullopt);
```

Comment:
Unrelated typo fix

Reply:
I took the minor opportunity to fix the typo within this patch, will undo if you have a strong concern on this.

Reply:
That was probably me. I vote for fix.